samples/dotnet/kernel-syntax-examples/Example22_OpenApiSkill_Jira.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Orchestration;
+using Microsoft.SemanticKernel.SkillDefinition;
+using Microsoft.SemanticKernel.Skills.OpenAPI.Authentication;
+using Newtonsoft.Json;
+using RepoUtils;
+
+public static class Example22_OpenApiSkill_Jira
```

Comment:
pls check https://github.com/microsoft/semantic-kernel/blob/main/samples/dotnet/kernel-syntax-examples/Example25_AADAuth.cs and https://github.com/microsoft/semantic-kernel/blob/main/samples/dotnet/kernel-syntax-examples/Example24_Tokenizer.cs. 

Could you add more information at the top explaining what the example is meant to show? Don't worry about being too verbose, people take these examples as learning docs